guides/source/active_record_querying.md

```diff
@@ -539,7 +539,7 @@ Client.order("orders_count ASC").order("created_at DESC")
 Selecting Specific Fields
 -------------------------
 
-By default, `Model.find` selects all the fields from the result set using `select *`.
+By default, finder queries will select all the fields from the result set using `select *`.
```

Comment:
Like above, I think this is confusing because "finder queries" is more confusing than an example. This change would be fine if we defined "finder queries" elsewhere, but for beginners I think changing the language here might be confusing.